source/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,95 @@
+.footer {
+  background: $color-black;
+  color: $color-white;
+  font-size: .9rem;
+
+  @include large() {
+    font-size: $font-size-base;
+  }
+
+  &__content {
+    border-bottom: $color-dark-gray solid .04rem;
+    margin: 0 auto;
+    padding: 2.3em 0;
+    width: 80%;
+
+    @include large() {
+      display: flex;
+      justify-content: space-between;
+      max-width: 68.75rem;
+    }
+  }
+
+  &__column {
+    flex: 1;
+    min-width: 10rem;
+
+    &--insiter {
+      flex: 3;
+    }
+
+    &--social {
+      max-width: 5rem;
+    }
+  }
+
+  &__section-title {
+    font-size: 1.125em;
+  }
+
+  &__logo {
+    color: $color-white;
+    font-family: $font-primary;
+    font-size: 3em;
+    font-weight: 700;
+    letter-spacing: -.03em;
+    margin-bottom: .3em;
+    margin-top: 0;
+
+    span {
+      color: $color-red;
+    }
+  }
+
+  &__usefull-links {
+    list-style: none;
+    padding: 0;
+
+    a {
+      color: $color-light-gray;
+      text-decoration: none;
+      transition: color .2s ease-out;
```

Comment:
Consider using a variable to default transition time.

Reply:
I removed some transitions that were "duplicated", and changed the transition on `svg` to be a little longer. So I don't think it's necessary to create a variable for the transition time.